test-framework/ui/src/main/java/org/keycloak/testframework/ui/page/AbstractPage.java

```diff
@@ -38,4 +40,14 @@ public boolean isActivePage() {
         return getExpectedPageId().equals(getCurrentPageId());
     }
 
+    public void assertCurrent() {
+        String name = getClass().getSimpleName();
+        Assertions.assertTrue(isCurrent(), "Expected " + name + " but was " + driver.getTitle() + " (" + driver.getCurrentUrl() + ")");
+    }
+
+    abstract public boolean isCurrent();
```

Comment:
Use `isActivePage` instead, and there's no need to implement this in extending classes, they just need to implement `getExpectedPageId` instead. You could maybe rename `isActivePage` to `isCurrent` and also add `assertCurrent()` so we don't need to change so much in migrated tests